saleor/plugins/avatax/__init__.py

```diff
@@ -237,23 +237,25 @@ def get_checkout_lines_data(
     ).filter(variant__product__charge_taxes=True)
     channel = checkout.channel
     for line in lines:
+        variant = line.variant
+        line_info = CheckoutLineInfo(
+            line=line,
+            variant=variant,
+            channel_listing=variant.channel_listings.get(channel=channel),
```

Comment:
This line will execute a database query for each iteration. We could fetch this data before entering the loop and store it in a dictionary for optimization. But I'm also wondering if we could use `fetch_checkout_lines` in this function instead of composing CheckoutLineInfo by hand.